build.zig

```diff
@@ -1,17 +1,18 @@
 const std = @import("std");
 
 pub fn build(b: *std.Build) void {
+    const target = b.standardTargetOptions(.{});
```

Comment:
LoadLibrary does only work on windows, so there's no point in offering multiple platforms